README.md

```diff
@@ -0,0 +1,114 @@
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+Hello-microverse is a repository created for educational purposes, to show tools and best practices a developer must master when creating files, branches and changes in a project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+VS Code editor
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML and CSS
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```

Comment:
[OPTIONAL] Please consider removing this placeholder from README. These placeholders are present in the template README to help to with wat to add in the README so that a user can easily interact with it ✨✨